### ISSUE TITLE
A request to add SwiftBlock as a third-party hosts manager in README

### ISSUE THREAD

**Issue by XploreInfinity** [StevenBlack/hosts#2001]
[SwiftBlock](https://github.com/xploreinfinity/swift-block) is a free and open source hosts based ad,malware and tracker blocker written in PyQt6 and Python. It ships with [unified hosts](https://github.com/StevenBlack/hosts) file from this repository, [AdAway default blocklist](https://adaway.org/hosts.txt) and  [Peter Lowe's hosts](https://pgl.yoyo.org/adservers/serverlist.php) by default. Additional hosts file sources can be added as per needs.

SwiftBlock is available as a [pypi package](https://pypi.org/project/swift-block/) for Linux, Windows, Mac OS and FreeBSD (Note: MacOS and FreeBSD platform support has been added,but not tested).

Features:

* Custom sources(You can easily add custom sources of hosts files)
* Custom Rules(You can manually redirect,allow or block specific hostnames)
* Easy access(Swift-Block will create a launcher/start menu shortcut on first run)

Caveats:

* Slightly complicated installation procedure for Windows Users(DNS Client service needs to be disabled)
* Single Threaded
* Requires running pip as root/superuser for installation

**Comment by StevenBlack** [comment]
Hi @XploreInfinity thank you for your question and request.

I would entertain a pull request for that.

You're aware that our list already includes `adaway.org` and `pgl.yoyo.org` lists, yes?

I don't think appropriating the `swift.org` is a good idea.  You don't need a logo, and you certainly don't need that logo.  That's my take 🤷🏻

**Comment by XploreInfinity** [comment]
> I would entertain a pull request for that.

Thank you very much. I will create one soon :)

> You're aware that our list already includes adaway.org and pgl.yoyo.org lists, yes?

Yes, however, since the project was inspired from Adaway, SwiftBlock by default comes with the hosts sources that Adaway ships with. 

> I don't think appropriating the swift.org is a good idea. You don't need a logo, and you certainly don't need that logo. That's my take 🤷🏻

You're probably right. However I do need a logo since swift-block is a GUI software(I'll need an app icon). I recently found out that Swift doesn't provide much freedom to use their logo outside of material relating to Swift. I will be replacing Swift's logo as soon as I find a viable replacement...

**Comment by XploreInfinity** [comment]
I apologize. I should have been more specific. Yes, I did make a mistake when I decided to use Swift's logo. And I am being unjust by continuing to use it. I can't immediately redact it, though.

 That's because doing so will break SwiftBlock, leaving anyone who decided to use it, confused. I can immediately replace it with a stub and add a new logo when I find one, but it is less complicated (both for me and the users) to switch directly to another logo which suits the application's needs.  When I said  'as soon as I find a viable replacement' I actually meant 2-3 days that I would require to find a new logo,test and push out an update for swiftblock.

I hope this clears things up. If you still consider my thought process irrational, I respect your decision to decline my request.
